.github/workflows/_ci.yaml

```diff
@@ -8,17 +8,22 @@ on:
         required: true
       BUILD_DATE:
         type: string
-        description: Build date in YYYY-MM-DD format
+        description: 'Build date in YYYY-MM-DD format'
         required: false
         default: NOT SPECIFIED
       MANIFEST_ARTIFACT_NAME:
         type: string
-        description: Artifact name in current run w/ manifest/patches. Leaving empty uses manifest/patches in current branch
+        description: 'Artifact name in current run w/ manifest/patches. Leaving empty uses manifest/patches in current branch'
         default: ''
         required: false
+      SOURCE_URLREFS:
+        type: string
+        description: 'A JSON object containing git url+refs for softwares to be built'
```

Comment:
nit: Should the description include that these are case-sensitive?

Reply:
Unlike `ci.yaml`, `_ci.yaml` is an internal workflow so I'm not too worried. Let me know if you think otherwise :grin: